src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -175,6 +189,13 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedValues(['ignore', 'ensure_single_line', 'ensure_fully_multiline'])
                 ->setDefault('ensure_fully_multiline')
                 ->getOption(),
+            (new FixerOptionBuilder(
+                'on_nested_multiline',
+                'When using ensure_fully_multiline, defines how to handle nested function arguments that contain newlines such as anonymous functions and arrays.'
+            ))
+                ->setAllowedValues(['ignore', 'ignore_single', 'always'])
```

Comment:
`ignore_single` is not very clear to me. I would go with a more explicit name such as `when_multiple_arguments`.